org-code-javabuilder/validation/src/main/java/org/code/validation/support/NeighborhoodTracker.java

```diff
@@ -58,11 +54,12 @@ public void trackEvent(ClientMessage message) {
     }
 
     if (!this.painterTrackers.containsKey(id) || !this.isInitialized) {
-      Logger.getLogger(MAIN_LOGGER)
-          .severe(
-              String.format(
-                  "Error: received a painter event for an uninitialized painter: %s, %s\n",
-                  message.getValue(), message.getDetail()));
+      String loggingString =
+          String.format(
+              "Error: received a painter event for an uninitialized painter: %s, %s\n",
+              message.getValue(), message.getDetail());
+      LoggerUtils.logSevereError(
+          (JavabuilderThrowableMessage) message, loggingString, new Throwable());
```

Comment:
i think i might need some java guidance on this one... originally, our severe-level error was just logging a string error. however, our current utils for severe errors all require a `Throwable cause` in some way:

https://github.com/code-dot-org/javabuilder/blob/main/org-code-javabuilder/protocol/src/main/java/org/code/protocol/LoggerUtils.java#L66-L98

a few questions about the code i have here:
1. i'm casting `message`, which is a `ClientMessage` to `JavabuilderThrowableMessage`, which extends `ClientMessage` -- is that appropriate here? (i'm a type-casting noob 😄)
2. i kept the original `loggingString` -- should it change now that we have other data being logged in our error?
3. is it okay that the `cause` will be `null`?

alternatively, i could add another util function that logs a severe error that just accepts a string 🤷🏻‍♀️

Reply:
Hmm, since LoggerUtils internally converts this data to a string, I think adding a new util method that logs a string is fine? But I'll also defer to @molly-moen on what would be the most helpful format here. Another option is to wrap the message in an `Exception` and just pass that in:
```
Exception e = new Exception(loggingString);
LoggerUtils.logSevereException(e);
```
Also another question for Molly...LoggerUtils.logSevere will publish a severe error metric. Should we be publishing errors for this case?

Reply:
For the formatting questions: I think for this case it would be most helpful to add a new severe error helper that doesn't require an exception, but instead gets passed a String `cause` and a String `message`. In our log parsing for the dashboard we use `cause` to aggregate errors and see patterns, so having a short `cause` for this case would be helpful. Then we can put any additional information into `message`.

Reply:
@sanchitmalhotra126 for this question: "LoggerUtils.logSevere will publish a severe error metric. Should we be publishing errors for this case?" Can we publish an error? My understanding is errors are an internal Lambda thing. Since this is coming from us I think it makes sense to be a severe error. This is sort of a weird one because the only way this would happen is if we had a bug in the painter code here which I don't think we do.

Reply:
Ah sorry my question was more around whether this particular situation warranted us publishing a severe error metric at all. Do we want to be tracking this/trigger an alarm for this? Agreed that it's highly unlikely, so it's probably fine.

Reply:
ah gotcha. I think at this point we could just get rid of this and ignore any unexpected events here. This should not ever happen.

Reply:
sorry, not sure i understand -- do we still want to log an error and publish a metric? or just log the error?

Reply:
We can just get rid of this log statement entirely, and just early return (right @molly-moen?)

Reply:
ah sorry I missed this message, yes we can just get rid of it and early return